websites/M/MUBI/presence.ts

```diff
@@ -16,6 +16,13 @@ async function getStrings() {
 		await presence.getSetting<string>("lang").catch(() => "en")
 	);
 }
+/* eslint-disable camelcase */
```

Comment:
Otherwise, the comment gets removed later.
```suggestion
/* eslint-disable camelcase */
const unused_variable = (a: number, b: number) => a + b;
unused_variable(1, 2);
```